R-packages/evalcast/R/scale_by_forecaster.R

```diff
@@ -0,0 +1,65 @@
+#' Scale error measures based on those of a particular forecaster.
+#'
+#' @param score_card score_card like that returned by `evaluate_predictions()`
+#' @param score_cols vector of column names in `score_card` to normalize
+#' @param base_forecaster_name name of forecaster in `score_card$forecaster` column by whose error
+#'   values the remaining forecasters' errors will be scaled
+#' @param id_cols vector of column names in `score_card` that identify distinct forecasts (i.e. the
+#'   independent variables of `score_card`).
+#' @param drop_base_entries whether to include the entries in `score_card` from 
```

Comment:
Suggest changing either `drop` or `include` to match the other.

Reply:
Done